pipeline_steps/tempest.groovy

```diff
@@ -25,11 +24,10 @@ def tempest_run(wrapper="") {
 /* if tempest install fails, don't bother trying to run or collect test results
  * however if running fails, we should still collect the failed results
  */
-def tempest(infra_vm=null, deploy_vm=null){
-  if (infra_vm != null) {
-    wrapper = "sudo ssh -T -oStrictHostKeyChecking=no ${infra_vm} \
-                RUN_TEMPEST_OPTS=\\\"${env.RUN_TEMPEST_OPTS}\\\" TESTR_OPTS=\\\"${env.TESTR_OPTS}\\\" "
-    copy_cmd = "scp -o StrictHostKeyChecking=no -p  -r infra1:"
+def tempest(deploy_vm=null){
+  if (deploy_vm != null) {
+    wrapper = "sudo ssh -T -oStrictHostKeyChecking=no ${deploy_vm}"
+    copy_cmd = "scp -o StrictHostKeyChecking=no -p  -r \$(${wrapper} ansible utility[0] -m command -a 'echo {{ physical_host }}')"
```

Comment:
This needs to have `cd /opt/rpc-openstack/rpcd/playbooks` before the ansible command.